tools/breaking-changes/data/breaking-changes.json

```diff
@@ -22948,7 +22948,7 @@
     ],
     "isDeleted": true,
     "deletedComment": "Class ClearCheckoutService has been removed and is no longer part of the public API.",
-    "migrationComment": "",
+    "migrationComment": "The processes are no longer needed, therefore the class was removed.",
```

Comment:
An alternative is to invalidate the CheckoutQuery (the details of the checkout data)

Reply:
Thanks, I mentioned the reset query event.